packages/trader/src/Stores/Modules/Trading/Constants/ui.js

```diff
@@ -1,7 +1,7 @@
-import Amount from 'Modules/Trading/Components/Form/TradeParams/amount.jsx';
-import Barrier from 'Modules/Trading/Components/Form/TradeParams/barrier.jsx';
-import Duration from 'Modules/Trading/Components/Form/TradeParams/Duration';
-import LastDigit from 'Modules/Trading/Components/Form/TradeParams/last-digit.jsx';
+import Amount from '../../../../Modules/Trading/Components/Form/TradeParams/amount.jsx';
+import Barrier from '../../../../Modules/Trading/Components/Form/TradeParams/barrier.jsx';
+import Duration from '../../../../Modules/Trading/Components/Form/TradeParams/Duration';
+import LastDigit from '../../../../Modules/Trading/Components/Form/TradeParams/last-digit.jsx';
```

Comment:
We shouldn't be using relative path this way. Please use alias

Reply:
sure :pray: